app/controllers/api/v1/accounts/csat_survey_responses_controller.rb

```diff
@@ -32,6 +32,7 @@ def set_csat_survey_responses
       Current.account.csat_survey_responses.includes([:conversation, :assigned_agent, :contact])
     )
     @csat_survey_responses = @csat_survey_responses.where(created_at: range) if range.present?
+    @csat_survey_responses = @csat_survey_responses.where(assigned_agent_id: params[:user_ids]) if params[:user_ids].present?
```

Comment:
Can you check if you can add scopes something like below in the CSAT model, for range filter and agent filter  
```
scope :filter_by_created_at, lambda { |range| where(created_at: range) if range.present? }
```

Reply:
Done